rts/lra-examples/coordinator-quarkus/pom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jboss.narayana.quickstart.rts.lra</groupId>
+    <artifactId>lra-examples</artifactId>
+    <version>5.13.1.Final-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>lra-coordinator-quarkus</artifactId>
+  <packaging>jar</packaging>
+  <name>Quarkus LRA Coordinator</name>
+
+  <properties>
+
+    <failOnMissingWebXml>false</failOnMissingWebXml>
+    <narayana.nodename.property>CoreEnvironmentBean.nodeIdentifier</narayana.nodename.property>
+    <narayana.version>${project.version}</narayana.version>
+    <quarkus.http.port>8080</quarkus.http.port>
+    <surefire-plugin.version>2.22.0</surefire-plugin.version>
+    <version.quarkus>2.12.1.Final</version.quarkus>
```

Comment:
There are 2.14 and 3.0 branches, are either of those options for us

Reply:
I will go with 2.14
I think 3.x version is the Jakarta one. It will go for the 6.x release